crates/sui-graphql-client/src/lib.rs

```diff
@@ -671,6 +673,44 @@ impl Client {
         }
     }
 
+    // ===========================================================================
+    // Package API
+    // ===========================================================================
+
+    // TODO: We should return maybe the MovePackage type, but the `data` field is not accessible.
+    // We might want to expose an API for getting the package or the struct depending what the
+    // object is.
+    /// Get a package's content by its address and optionally, a specific version.
+    pub async fn package(
+        &self,
+        address: Address,
+        version: Option<u64>,
+    ) -> Result<Option<Object>, Error> {
```

Comment:
What value does this have over just a normal object lookup since its just returning an Object anyway?

Reply:
I was hoping we can return a move package instead, but it seems that the BCS returned by GraphQL is the object. So indeed, no need for it.

Reply:
i would think there would be a way to have graphql return the normalized package